configure.ac

```diff
@@ -766,7 +773,7 @@ echo "|    Razor AHRS .......................... : $have_razor"
 echo "|    VRPN ................................ : $have_vrpn"
 echo "|"
 echo "| Build with Ecasound support ............ : $have_ecasound"
-echo "| Build with IP interface ................ : $have_ip_interface"
+echo "| Network: legacy/WebSocket .............. : $have_ip_interface/$have_websocket_interface"
```

Comment:
Shouldn't this be two separate things?

Reply:
They are two separate things, just on the same line.

I wanted to keep the number of lines low, so that the summary is fully visible on a 25-line terminal.

But probably that's not important ...?

Would you prefer two separate lines?

Reply:
Ah, now I get it. Probably was a little tired reading this line.
It makes sense and should be extended for the OSC interface (earliest after LAC... time's to scarce right now).